scripts/ubuntu_setup.sh

```diff
@@ -0,0 +1,2 @@
+sudo apt-get update
+sudo apt-get install libxkbcommon-dev libwayland-dev
```

Comment:
I had to install additional libraries on WSL:
```suggestion
sudo apt-get install libxkbcommon-dev libwayland-dev xorg-dev libglu1-mesa-dev pkg-config libx11-dev libxrandr-dev
```